### PR TITLE
added theme support for clocks and small fixes

### PR DESCRIPTION
**Pull request by mymike00**
Adds the SuruDark theme support for clocks
fixed big text in information page
improve UX for the swipe left/right gesture to change page

### CODE REVIEW

**Comment by dark-eye** [comment]
Thank you very much  @mymike00  :+1: 
I`ll try and have a look on it tonight.

**Comment by mymike00** [comment]
Great!
another pr will arrive in the next days for live theme switching, actually woeking but still a bit buggy

**Comment by dark-eye** [comment]
@mymike00 It look very cool !

I`ll post it to the QA group if there's no new issues  and the big bosses (@Flohack74 , @mariogrip , @UniversalSuperBox  ) will approve the visible change below :
screenshot20181025_101625993
screenshot20181025_101622399
screenshot20181025_101619828


**Compiled  version** : [here](https://my.owndrive.com/index.php/s/6Q6FJnr04UbQjRQ)

 i`ll merge it as soon as i can.

File: app/components/AdjustableAnalogClock.qml [hunk 1]
```diff
@@ -97,7 +97,7 @@ AnalogMode {
         anchors.centerIn: parent
 
         smooth: true
-        source: "../graphics/Minute_Adjust_Cirlce.png"
+        source: Theme.name == "Ubuntu.Components.Themes.Ambiance" ? "../graphics/Minute_Adjust_Cirlce.png" : "../graphics/Minute_Adjust_Cirlce_White.png"
```
Comment (dark-eye) [inline app/components/AdjustableAnalogClock.qml]: It  might  be better to use a lookup table/hash to match the  theme to the associated graphics  but  i don't  think this should hold up the merge.

Comment (mymike00) [reply app/components/AdjustableAnalogClock.qml]: I thought also about adding a filter/shader on that image as an `Icon` would do to change to color from grey to white (#f2f2f2) but It wasn't so easy...

Comment (mymike00) [reply app/components/AdjustableAnalogClock.qml]: Btw, the telegram app (the actual one, not the newest) uses similar code for some theming stuff and imo is a quite good code, easy and efficient

**Comment by dark-eye** [comment]
Anyone has any misgiving about merging this new design?

**Comment by mymike00** [comment]
> Some of the tertiary text might be secondary

@cibersheep do you mean the city names, the date, the position and the stuffs in the first screenshot?
those label weren't themable, I fixed it in #105 
screenshot20181109_082100990

**Comment by cibersheep** [comment]
Nice! Then @bhdouglass I would say: let's gooooo :)

**Comment by bhdouglass** [comment]
@dark-eye go for it :)

**Pull request merged**